src/NHibernate/SelectMode.cs

```diff
@@ -33,12 +33,12 @@ public enum SelectMode
 		JoinOnly,
 
 		/// <summary>
-		/// Skips fetching for eagerly mapped association (no-op for lazy association).
```

Comment:
It won't skip  `lazy="false"` association from loading - so better say it explicitly which case is handled here